src/shared/types/index.ts

```diff
@@ -56,12 +57,13 @@ interface LedgerResponseCorrected extends LedgerResponse {
   >
 }
 
-interface AmendmentEnabled {
+interface AmendmentStatus {
```

Comment:
A docstring explaining what this represents would be helpful. 

Previously this just represented amendments that were enabled on a network.

Now it's representing two things:
1. An enabled amendment
2. An amendment which is being voted in. 

A different way you could represent that which might be more accurate would be 

```

interface AmendmentEnabled {
  amendment_id: string
  networks?: string
  ledger_index?: number
  tx_hash: string
  date?: Date
}

interface AmendmentPending extends AmendmentEnabled {
  tx_hash?: string
  eta?: Date
}

type AmendmentStatus = AmendmentEnabled | AmendmentPending
```

Reply:
`tx_hash` should be optional on `AmendmentEnabled` too as we might not know when it was enabled on some networks such as if history was lost or it was activated at genesis.